### ISSUE TITLE
Adding a way to reference children from parent

### ISSUE THREAD

**Issue by valentinradu** [schibsted/layout#116]
If I'm not mistaken, right now there is no way to get the content between the tags and use it in another manner than direct children. This makes things a bit difficult when wanting to manipulate your children inside the parent. Would this be a desirable feature? Is there a way to do this right now?

```
<UIView
xml="Panel.xml">
  <UIStackView
    width="100% - 40" 
    top="{previous.bottom + 40}"
    center.x="{parent.center.x}"
    axis="vertical"
    spacing="20">
     <Children/>
  </UIStackView>
</UIView>
```

**Comment by nicklockwood** [comment]
You’re not mistaken - there’s no way to do this currently. This looks like a good solution though.

**Comment by nicklockwood** [comment]
@valentinradu I've added support for a `<children/>` tag in templates. It's on the [develop](https://github.com/schibsted/layout/branches) branch if you want to try it out.

**Comment by valentinradu** [comment]
Wow, that's awesome. Wanted to implement it myself over the weekend since I couldn't find the time this week. But thank you, I'll check it out!

**Comment by nicklockwood** [comment]
@valentinradu OK, this has landed in version 0.6.21
